nss-evening-cohort-14/SweetTooth
# Mg admin page

## Description

fleshed out admin page to show users, moods, snacks, and orders

## Related Issue

#69 

## Motivation and Context
basic skeleton for admin user to see all users, moods, snacks, orders

## How Can This Be Tested?
git fetch mg-admin-page
run back and front end
auth yourself so you can see orders
visit admin page

## Screenshots (if appropriate):

## Types of changes
